### ISSUE TITLE
postgresqlPassword is required when using externalPostgreSQL

### ISSUE THREAD

**Issue by alionar** [getredash/contrib-helm-chart#63]
When checking the generated yaml file with dry run:
`helm install redash_ori --create-namespace  -n redash -f redash_values.yaml --debug --dry-run redash/redash`

i got this error when using `externalPostgreSQL` and `postgresql.enable: false`
```install.go:172: [debug] Original chart version: ""
install.go:189: [debug] CHART PATH: /home/ec2-user/.cache/helm/repository/redash-2.0.0.tgz

Error: values don't meet the specifications of the schema(s) in the following chart(s):
postgresql:
- postgresqlPassword: Invalid type. Expected: string, given: null

helm.go:81: [debug] values don't meet the specifications of the schema(s) in the following chart(s):
postgresql:
- postgresqlPassword: Invalid type. Expected: string, given: null```

**Comment by alionar** [comment]
redash_value.yaml
```image:
  repository: redash/redash
  tag: 7.0.0.b18042

env:
  TZ: WIB-07
  PYTHONUNBUFFERED: 0

nameOverride: "redash_ori"

redash:
  googleClientId: my_client_id
  googleClientSecret: $(REDASH_GOOGLE_CLIENT_SECRET)
  mailServer: aspmx.l.google.com
  mailPort: 25
  mailDefaultSender: my@example.com
  webWorkers: 8
  dateFormat: 'YYYY/MM/DD'
  featureAllowCustomJsVisualizations: true
  logLevel: INFO
  cookieSecret: $(REDASH_COOKIE_SECRET)
  secretKey: $(REDASH_SECRET_KEY)
  host: https://test.example.com

adhocWorker:
  env:
    QUEUES: "queries"
    WORKERS_COUNT: 4

scheduledWorker:
  env:
    QUEUES: "scheduled_queries"
    WORKERS_COUNT: 2

postgresql:
  enable: false

externalPostgreSQL: postgresql://redash:$(POSTGRESQL_PASSWORD)@host:5432/redash_2019_eks_test

envSecretName: redash-secret```

**Comment by grugnog** [comment]
I believe this was just resolved by #57 57

**Comment by Tarasovych** [comment]
Still an issue with 3.0.1
